### PR TITLE
7.19b changes so Travis shuts up

### PR DESCRIPTION
**Pull request by DarkoniusXNG**
I didn't include nerfs for heroes that don't deserve nerfs in OAA.
http://cyborgmatt.com/719b/

### CODE REVIEW

File: game/scripts/npc/items/item_ring_of_aquila.txt [hunk 1]
```diff
@@ -66,7 +66,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "10 20 40"
+        "bonus_damage"                                    "10 20 40" //OAA
```
Comment (chrisinajar) [inline game/scripts/npc/items/item_ring_of_aquila.txt]: match dota here

File: game/scripts/npc/abilities/tiny_craggy_exterior.txt [hunk 2]
```diff
@@ -30,7 +30,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"       "20 30 40 50 110 170"
+    "AbilityManaCost"       "20 30 40 50 110 170" //OAA
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/tiny_craggy_exterior.txt]: match dota here

File: game/scripts/npc/abilities/tiny_avalanche.txt [hunk 3]
```diff
@@ -25,7 +25,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "20 19 18 17"
+    "AbilityCooldown"                                     "20 19 18 17" //OAA
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/tiny_avalanche.txt]: match dota here

File: game/scripts/npc/abilities/silencer_curse_of_the_silent.txt [hunk 4]
```diff
@@ -30,7 +30,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "75 95 115 135 235 335"
+    "AbilityManaCost"                                     "75 95 115 135 235 335" //OAA
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/silencer_curse_of_the_silent.txt]: match dota here

File: game/scripts/npc/abilities/crystal_maiden_crystal_nova.txt [hunk 5]
```diff
@@ -30,7 +30,7 @@
 
     // Cost
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityManaCost"                                     "130 140 150 160 200 240"
+    "AbilityManaCost"                                     "130 140 150 170 200 240" //OAA
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/crystal_maiden_crystal_nova.txt]: you made it 5 mana different at max level..... just match dota here....

Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/crystal_maiden_crystal_nova.txt]: ah come on Chris, nobody plays CM in OAA

Comment (DarkoniusXNG) [reply game/scripts/npc/abilities/crystal_maiden_crystal_nova.txt]: I see, its 5 mana difference, nvm

**Comment by DarkoniusXNG** [comment]
Made the requested changes.

**Pull request merged**